.github/workflows/pythonapp.yml

```diff
@@ -8,33 +8,34 @@ env:
 jobs:
   analysis:
     name: Code Analysis
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
       - name: Set up Python 3.8
         uses: actions/setup-python@v1
         with:
           python-version: 3.8
-      - name: Install lint & packaging dependencies.
+      - name: Install project dependencies
         run: |
           python -m pip install -U pip --no-cache-dir
           python -m pip install -e . -r dev_requirements.txt
       - name: Lint
         run: invoke lint
   tests:
     name: Tests
-    runs-on: ubuntu-18.04
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
         python-version: [3.6, 3.7, 3.8]
-      fail-fast: true
+      fail-fast: false
```

Comment:
Why change the `fail-fast` argument?

Reply:
More information to see if the failures are OS/version specific